examples/cpp/aot_inductor/llama2/compile.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+
+import torch
+import torch._export
+from model import ModelArgs, Transformer
+
+
+def load_checkpoint(checkpoint):
+    # load the provided model checkpoint
+    checkpoint_dict = torch.load(checkpoint, map_location="cpu")
+    gptconf = ModelArgs(**checkpoint_dict["model_args"])
+    model = Transformer(gptconf)
+    state_dict = checkpoint_dict["model"]
+    unwanted_prefix = "_orig_mod."
+    for k, v in list(state_dict.items()):
+        if k.startswith(unwanted_prefix):
+            state_dict[k[len(unwanted_prefix) :]] = state_dict.pop(k)
+    model.load_state_dict(state_dict, strict=False)
+    model.eval()
+    return model, gptconf
+
+
+if __name__ == "__main__":
```

Comment:
it seems that this script can be used to support a list of models. Can we move it to   an utils dir?

Reply:
kind of, the file needs to be adjusted for different models and inputs (see constraints). We can see later if we can leverage a common structure. For now this is specific to this (kind of) model.